src/store/index.ts

```diff
@@ -47,19 +55,30 @@ const {
     setWorkspaces(state, workspaces: string[]) {
       state.workspaces = workspaces;
     },
-    setCurrentWorkspace(state, workspace: WorkspaceSpec) {
+
+    setCurrentWorkspace(state, workspace: WorkspaceState) {
       state.currentWorkspace = workspace;
     },
+
+    setEmptyWorkspace(state, name: string) {
```

Comment:
This would be better named as `unsetWorkspace` IMO.

Reply:
And there shouldn't be a `name` argument, right? Isn't the point of this to set the workspace to `null`? Or did I miss something?

Reply:
You're right, I was under the impression that we needed the `name` property on the workspace to correctly display it, but it's tied to the route. I'll make that change. 

Also if it wasn't clear, the point of this mutation is to have it unset when switching between workspaces, so that the tables/networks of one don't bleed over into the other when making the transition.

Reply:
See 3af1a34, also I know you advocated for `unsetWorkspace`, but since the reciprocal call is `setCurrentWorkspace`, I changed it to `unsetCurrentWorkspace`